charts/sn-platform/values.yaml

```diff
@@ -208,6 +219,59 @@ images:
     repository: k8s.gcr.io/prometheus-adapter/prometheus-adapter
     tag: "v0.9.0"
     pullPolicy: IfNotPresent
+  pulsar_metadata:
+    repository: streamnative/sn-platform
+    tag: "2.8.1.4"
+    pullPolicy: IfNotPresent
+  configmapReload:
+    repository: jimmidyson/configmap-reload
+    tag: v0.3.0
+    pullPolicy: IfNotPresent
+  external_dns:
+    repository: k8s.gcr.io/external-dns/external-dns
+    tag: "0.7.3"
+    pullPolicy: IfNotPresent
+  imagePuller:
+    hook:
+      repository: streamnative/k8s-image-awaiter
+      tag: "0.1.0"
+      pullPolicy: IfNotPresent
+    pause:
+      repository: gcr.io/google_containers/pause
+      tag: "0.3.0"
+      pullPolicy: IfNotPresent
+  functionnmesh:
+    repository: streamnative/function-mesh
+    tag: v0.1.6
+    pullPolicy: IfNotPresent
+  pulsar_operator:
+    repository: streamnative/pulsar-operator
+    tag: v0.8.15
+    pullPolicy: IfNotPresent
+  bookkeeper_operator:
+    repository: streamnative/pulsar-operator
+    tag: v0.8.15
+    pullPolicy: IfNotPresent
+  zookeeper_operator:
+    repository: streamnative/pulsar-operator
+    tag: v0.8.15
+    pullPolicy: IfNotPresent
+  cert_manager_controller:
+    repository: quay.io/jetstack/cert-manager-controller
+    tag: v1.3.1
+    pullPolicy: IfNotPresent
+  cert_manager_cainjector:
+    repository: quay.io/jetstack/cert-manager-controller
+    tag: v1.3.1
+    pullPolicy: IfNotPresent
+  cert_manager_webhook:
+    repository: quay.io/jetstack/cert-manager-controller
+    tag: v1.3.1
+    pullPolicy: IfNotPresent
+  vault_operator:
+    repository: ghcr.io/banzaicloud/vault-operator
+    tag: v1.3.0
+    pullPolicy: IfNotPresent
```

Comment:
@tuteng Adding images for cert-manager, pulsar-operator and function-mesh in this values doesn't appear to work. There is no template related in sn-platform chart.

Reply:
Yes, this section just puts all the image needed in snp here, so that users can easily find all the image used in snp

Reply:
I see, but it will be quite confusing for users. I think it will be better if we put the images list on Platform docs.